### DISCUSSION TITLE
Variated Pumpkin’s

### DISCUSSION THREAD

**Discussion by Twiistz4264** [BedrockTweaks/Files#363] [Ideas]
Just like the title says it would randomize pumpkins as an example white pumpkins or different faces different color shading etc would apply to all current pumpkin types aka uncarved carved and jack O lanterns!

Also I understand this probably would take some times due to other things being worked on aka (addons)coming soon

**Comment by SquatchHunter** [comment]
community packs will be done after addons are
we still dont have a timeline
